Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -426,74 +428,13 @@ public void StartClient(Uri uri)
             OnStartClient();
         }
 
-        void StartHostClient()
-        {
-            if (LogFilter.Debug) Debug.Log("NetworkManager ConnectLocalClient");
-
-            if (authenticator != null)
-            {
-                authenticator.OnStartClient();
-                authenticator.OnClientAuthenticated.AddListener(OnClientAuthenticated);
-            }
-
-            networkAddress = "localhost";
-            NetworkServer.ActivateLocalClientScene();
-            RegisterClientMessages();
-
-            // ConnectLocalServer needs to be called AFTER RegisterClientMessages
-            // (https://github.com/vis2k/Mirror/pull/1249/)
-            NetworkClient.ConnectLocalServer();
-
-            OnStartClient();
-        }
-
         // FinishStartHost is guaranteed to be called after the host server was
         // fully started and all the asynchronous StartHost magic is finished
         // (= scene loading), or immediately if there was no asynchronous magic.
         //
         // note: we don't really need FinishStartClient/FinishStartServer. the
         //       host version is enough.
         bool finishStartHostPending;
```

Comment:
might want to move this one above finishstarthost again

Reply:
I put it there because it's first used in `finishStartHostPending` immediately after this declaration.

Reply:
check out the comment above. it's really part of FinishStartHost (including the comment) to understand how it works :)

Reply:
Moved and added comments...better?  I really don't like a declaration between the two methods that use it.

Reply:
just keep it above finishstarthost mate. I put it there for a reason :))

Reply:
Done